src/components/your_grants/view_applicants/table/index.tsx

```diff
@@ -11,14 +11,18 @@ function Table({
   onManageApplicationClick,
   data,
   title,
+  isReviewer,
   archiveGrantComponent,
+  applicantionReviewer,
 }: {
   onViewApplicantFormClick?: (data? : any) => void;
   // onAcceptApplicationClick?: () => void;
   // onRejectApplicationClick?: () => void;
   onManageApplicationClick?: (data? : any) => void;
   data: any[];
   title: string;
+  isReviewer : boolean;
+  applicantionReviewer: any[];
```

Comment:
No more usage of "any" in the code please

Reply:
fix